### DISCUSSION TITLE
Authroizatioon issue

### DISCUSSION THREAD

**Discussion by DeveloperGiniLytics** [jasontaylordev/CleanArchitecture#889] [Q&A]
I am using an Identity server with a clean architecture but I having an error in API When I log in from UI Side and didn't pass any token in API it is giving me the result in API and this behavior is not excepted How can I fix this 
![image](https://github.com/jasontaylordev/CleanArchitecture/assets/101571737/38999342-4e9b-42a1-8cce-eb9ab048c083)

**Comment by jasontaylordev** [comment]
Apologies, I cannot provide support for Identity Server.
Also, please note that from the .NET 8 version the template no longer uses Identity Server.
Thanks.
